src/contracts/Repository/Events/Content/BeforeLoadContentEvent.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Repository\Events\Content;
+
+use Ibexa\Contracts\Core\Repository\Event\BeforeEvent;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use UnexpectedValueException;
+
+final class BeforeLoadContentEvent extends BeforeEvent
+{
+    private int $contentId;
+
+    /** @var string[]|null */
+    private ?array $languages;
+
+    private ?int $versionNo;
+
+    private bool $useAlwaysAvailable;
+
+    private ?Content $content = null;
+
+    /**
+     * @param string[] $languages
+     */
+    public function __construct(
+        int $contentId,
+        array $languages = null,
```

Comment:
```suggestion
        ?array $languages = null,
```
https://wiki.php.net/rfc/deprecate-implicitly-nullable-types